include/openenclave/internal/evidence.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/**
+ * @file evidence.h
+ *
+ * This file defines object identities for SGX evidence generation and
```

Comment:
The comment says "SGX" but this file is not SGX specific.  (If it were, it would need sgx in the filename or directory name)
```suggestion
 * This file defines object identities for evidence generation and
```